features/steps/test_starter/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/__main__.py

```diff
@@ -1,45 +1,16 @@
 """{{ cookiecutter.project_name }} file for ensuring the package is executable
 as `{{ cookiecutter.repo_name }}` and `python -m {{ cookiecutter.python_package }}`
 """
-import importlib
 from pathlib import Path
 
-from kedro.framework.cli.utils import KedroCliError, load_entry_points
+from kedro.framework.cli.project import find_run_command
```

Comment:
This increases the API surface, so we should be mindful adding it here. If we are making it a public API, then I'd rather put it in `cli.utils`.

Reply:
Fair point. As I mentioned earlier, I didn't intent this to be public (https://github.com/kedro-org/kedro/pull/4012#discussion_r1688063853) If it's moved to `cli.utils` though I can bring back a `run` method that calls `find_run_command`.

Reply:
```
from kedro.framework.cli.project import find_run_command
```
If it's used here, then it becomes public anyway. If we'd like to import the run, then maybe it makes sense to be put in `kedro.framework.cli.project`, else if it's the function `find_run_command`, it fits better under `kedro.framework.cli.utils` imo. I think both would be great improvements over the current state anyways.

Reply:
I've moved `find_run_command` to cli utils.